Wikipedia/Code/HintViewController.swift

```diff
@@ -114,9 +117,11 @@ extension HintViewController: Themeable {
         guard viewIfLoaded != nil else {
             return
         }
-        view.backgroundColor = theme.colors.hintBackground
-        defaultLabel?.textColor = theme.colors.link
-        confirmationLabel?.textColor = theme.colors.link
-        confirmationAccessoryButton.tintColor = theme.colors.link
+
+        view.backgroundColor = backgroundColor ?? theme.colors.hintBackground
```

Comment:
I use it when the same view can have different colors coming from `Theme` - for example, in `WMFEmptyView`, one of the colors can be swapped for a different `Theme` color: https://github.com/wikimedia/wikipedia-ios/blob/54ea08c2aaf85c39e383a6532cf2a046a5ef3347/Wikipedia/Code/WMFEmptyView.m#L261

Here it wouldn't work because the colors don't have to be `Theme` colors - for example, the background color set by `PageHIstoryHintController` is `UIColor.wmf_lightRed`. If it changes, it would make sense to use key paths